ddl/ddl.go

```diff
@@ -1198,6 +1198,10 @@ func (d *ddl) SwitchConcurrentDDL(toConcurrentDDL bool) error {
 		})
 	}
 
+	if variable.EnableMDL.Load() && !toConcurrentDDL {
+		return errors.New("can not disable concurrent ddl when metadata lock is enabled")
```

Comment:
Do we have a place to collect all error msg?

Reply:
For standard error with the error code, yes. For these misc errors, no.